src/Integration/Binding/ProjectRuleSetWriter.cs

```diff
@@ -74,10 +79,17 @@ private bool TryUpdateExistingProjectRuleSet(string solutionRuleSetPath, string
             }
 
             existingRuleSetPath = PathHelper.ResolveRelativePath(currentRuleSet, projectRuleSetRootFolder);
+
```

Comment:
No, PathHelper uses `Uri` internally which will canonicalise them first.
But I've also added a test case to cover this fact; `PathHelper_CalculateRelativePath`.